runtime/j9vm/java11vmi.c

```diff
@@ -1127,19 +1303,45 @@ JVM_AddModulePackage(JNIEnv * env, jobject module, const char *package)
  * 2) module is unnamed or
  * 3) package is not in module
  */
+#if JAVA_SPEC_VERSION >= 15
+void JNICALL
+JVM_AddModuleExportsToAllUnnamed(JNIEnv * env, jobject fromModule, jstring packageObj)
+#else
 void JNICALL
 JVM_AddModuleExportsToAllUnnamed(JNIEnv * env, jobject fromModule, const char *package)
+#endif /* JAVA_SPEC_VERSION >= 15 */
 {
 	J9VMThread * const currentThread = (J9VMThread*)env;
 	J9JavaVM const * const vm = currentThread->javaVM;
 	J9InternalVMFunctions const * const vmFuncs = vm->internalVMFunctions;
+#if JAVA_SPEC_VERSION >= 15
+	jboolean isCopy = JNI_FALSE;
+	const char* package = (NULL == packageObj) ? NULL : (*env)->GetStringUTFChars(env, packageObj, &isCopy);
+	if (JNI_FALSE == isCopy) {
+		Assert_SC_unreachable();
+		package = NULL;
```

Comment:
I'm not sure of the point of clearing `package`; if it were non-NULL, it represents a leak, but the preceding assertion means we don't get here. (Likewise for previous occurrences of this pattern.)

Reply:
Assertions can be disabled, I thought there should be some semi-reasonable path if they are. Clearing `package` will cause an exception to be thrown.

Reply:
If we wanted to be pedantic, we could say:
```
(*env)->ReleaseStringUTFChars(env, packageObj, package);
```
before nulling the pointer.